dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -226,8 +226,19 @@ public static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceC
      * @param configPath
      */
     private static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceConfiguration> support, String configPath) {
-        List<String> migrations = List.of("1.3.0.generated", "1.3.1.consistency", "test.confidential1", "1.4.0", "1.5.0", "test.confidential1_1.5.0", "1.6.0", "1.7.0", "1.8.0", "1.9.0", "1.10.0", "1.11.0", "1.12.0", "1.13.0");
-        dropAllAndRunMigration(migrations, support.getApplication(), configPath);
+        dropAllAndRunMigration(listMigrations("test.confidential1", "test.confidential1_1.5.0"), support.getApplication(), configPath);
+    }
+
+    /**
+     * Returns a list of migrations containing COMMON_MIGRATIONS and additional migrations
+     * @param additionals
+     * @return
+     */
+    public static List<String> listMigrations(String... additionals) {
+        if (additionals.length > 0) {
+            return Stream.concat(COMMON_MIGRATIONS.stream(), Stream.of(additionals)).collect(Collectors.toList());
```

Comment:
Entirely out of curiosity, and no need to change this, but could this line do all the work by itself, no `if` necessary?

Reply:
Yeah you're right, the `if` isn't necessary